float64 angleRot
float64 angleLeft
float64 angleRight
---
bool success
